### PR TITLE
Fix CI setup

### PR DESCRIPTION
**Pull request by bitPogo**
### :pushpin: References
n/a

### :tophat: What is the goal?

Change Actions to cloud runner instead of self hosted.

### :unicorn: How is it being implemented?

n/a

Bump Gradle 6.9.1 -> 7.4.1

### :thinking: DOD Checklist

- [x] My code follows the code style and naming conventions of this project.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have updated the documentation.
- [ ] I have updated the changelog accordingly.

### CODE REVIEW

File: .github/workflows/d4l-ci-latest-version.yml [hunk 1]
```diff
@@ -29,10 +31,10 @@ jobs:
         run: adb install ./script/ci/android/com.android.chrome_79.0.3945.93-394509315-x86-nodpi.apk
       - name: Cleanup
         run: ./gradlew clean
```
Comment (bitPogo) [inline .github/workflows/d4l-ci-latest-version.yml:33]: This could go as well

Comment (wmontwe) [reply .github/workflows/d4l-ci-latest-version.yml]: Updated

File: .github/workflows/d4l-ci-publish-release.yml [hunk 2]
```diff
@@ -19,17 +23,33 @@ jobs:
...
+        with:
+          api-level: ${{ matrix.api-level }}
+          target: google_apis
+          arch: x86_64
+          profile: Nexus 5
+          force-avd-creation: false
+          avd-name: d4l-sdk-${{ matrix.api-level }}
+          emulator-options: -no-snapshot -no-audio -no-boot-anim -camera-back none
+          disable-animations: true
+          script: ./gradlew --stacktrace :sdk-android:test
```
[the remark this answers is not in the document]
Comment (wmontwe) [reply .github/workflows/d4l-ci-publish-release.yml:52]: yes, otherwise the Gradle error would not have shown up

**Review by wmontwe — APPROVED** [review]
👍

**Pull request merged**